convert_ti_simplelink_academy_01_11_00_0000_to_linux.sh

```diff
@@ -25,14 +26,17 @@ fi
 
 TI_ROOT_DIRECTORY=$1
 
-BLE_SDK_VERSION=01_11_00_0000
+SIMPLELINK_SDK_VERSION=01_11_00_0000
+BLE_SDK_VERSION=ble_sdk_2_02_01_18
 TIRTOS_VERSION=tirtos_cc13xx_cc26xx_2_20_01_08
 XDCTOOLS_VERSION=xdctools_3_32_00_06_core
 XDCTOOLS_SOURCE=http://software-dl.ti.com/dsps/dsps_public_sw/sdo_sb/targetcontent/rtsc/3_32_00_06/index_FDS.html
 
 WINE_ROOT=~/.wine/drive_c/ti
 
-BLE_SDK_DIRECTORY=${TI_ROOT_DIRECTORY}/simplelink_academy_${BLE_SDK_VERSION}/modules
+SIMPLELINK_SDK_DIRECTORY=${TI_ROOT_DIRECTORY}/simplelink_academy_${SIMPLELINK_SDK_VERSION}
+BLE_SDK_DIRECTORY=${TI_ROOT_DIRECTORY}/simplelink/${BLE_SDK_VERSION}
+SIMPLELINK_SDK_MODULES_DIRECTORY=${TI_ROOT_DIRECTORY}/simplelink_academy_${SIMPLELINK_SDK_VERSION}/modules
```

Comment:
Shouldn't this line use ${SIMPLELINK_SDK_DIRECTORY}

Reply:
Thanks for your review.
I have added a new commit that fix the SIMPLELINK_SDK_MODULES_DIRECTORY variable